src/analytics/Properties.tsx

```diff
@@ -1211,6 +1212,13 @@ interface TokenBottomSheetEventsProperties {
   }
 }
 
+interface InAppReviewEventsProperties {
```

Comment:
i think that this can be part of AppEventsProperties right?

Reply:
Fixed in 24e81d31a615d6f02a97c876293433aee9eae3c6!